### PR TITLE
Bracelet Clothing

### PR DESCRIPTION
**Pull request by TheScarFr**
Add the "Bracelet" Clothing Item by default in the wardrobe:

FemaleBracelet

MaleBracelet

[Changes.txt](https://github.com/user-attachments/files/17607866/Changes.txt)

### CODE REVIEW

File: compiled/dat/ACASpanish.loc [hunk 1]
```diff
@@ -28,6 +28,9 @@
 			<element name="Boots">
 				<translation language="Spanish">Botas</translation>
 			</element>
+			<element name="Bracelet">
+				<translation language="Spanish">Brazalete</translation>
```
Comment (Hoikas) [inline compiled/dat/ACASpanish.loc:32]:
```suggestion
				<translation language="Spanish">Pulsera</translation>
```

El brazalete is an ornate wrist accessory. The bracelet you're submitting appears to be an ordinary wristband, so la pulsera is more appropriate.

**Review by Hoikas — COMMENTED** [review]
A few concerns:
- Please only submit translations that have been done by a human. If you don't know the language and aren't able to get a real person to submit the translation, leave it blank for someone who knows the language to translate.
- We don't need to have PNGs, targas, and DDS files for the images. The source file and a rasterized copy, ideally PNG, is sufficient.

**Comment by Hoikas** [comment]
Good - we need to make sure they all get credited as co-authors on the commit.

**Comment by dgelessus** [comment]
An alternative way to jump to the message - perhaps it works better for you:

1. Copy the Discord message link
2. Open Discord the regular way
3. Go to your DMs/friends page
4. Click on the "Find or start a conversation" search box
5. Paste in the Discord message link
6. Press Enter (or click on the "Go to message" entry that appears in the "search" results)

**Comment by Hoikas** [comment]
That still doesn't seem to work, I'm afraid:

image

**Comment by dgelessus** [comment]
Well that's disappointing... The correct message is roughly 10 messages after the last message shown on your screenshot.

**Comment by TheScarFr** [comment]
Is there something that I must do?

**Comment by TheScarFr** [comment]
The Bracelet Clothing Item have been added as is to the MOUL Staging Server.
Should I commit anyway the suggestion that you made, @Hoikas ?

**Comment by Hoikas** [comment]
We need to:
- Amend the first commit [775fe26](https://github.com/H-uru/moul-assets/pull/265/commits/775fe26d6bc8d44c489708480654a895898f3b98) with authorship information for the translators
- Correct the Spanish translation, probably in a separate commit.

I can do those for you if you can provide the names *and* email addresses of the people who did the translation.

**Comment by dgelessus** [comment]
In my case, please use `dgelessus <dgelessus@users.noreply.github.com>` for the co-author information, like for all my other commits here.

For the other contributors - if we don't have a Git author email address for them, and they would prefer to not have an email address published, I assume crediting just by their name would be sufficient too, right?

**Comment by Hoikas** [comment]
> For the other contributors - if we don't have a Git author email address for them, and they would prefer to not have an email address published, I assume crediting just by their name would be sufficient too, right?

I think that will work.

**Comment by TheScarFr** [comment]
> We need to:
> 
>     * Amend the first commit [775fe26](https://github.com/H-uru/moul-assets/pull/265/commits/775fe26d6bc8d44c489708480654a895898f3b98) with authorship information for the translators
> 
>     * Correct the Spanish translation, probably in a separate commit.
> 
> 
> I can do those for you if you can provide the names _and_ email addresses of the people who did the translation.

I don't find how to "Amend" the commit in GitHub nor in Fork...
If you could do it for me, please, it would be nice.
Here is what is on Stagging:
```
[new]    New Wardrobe Item added. {dgelessus,Ehren,Riccardo Salerano,Salvo Ludus,Sirius,TheScar(.fr)}
```
More details:
```
- Development:
	- Ehren
	- Sirius
	- TheScar(.fr)

- Localisations:
	- English: TheScar(.fr)
	- French: TheScar(.fr)
	- German: dgelessus <dgelessus@users.noreply.github.com>
	- Italian: Riccardo Salerano
	- Spanish: Salvo Ludus
```
Why are the email addresses needed?

For the Spanish translation, If you want to replace "Brazalete" by "Pulsera", you will have to replace "Salvo Ludus" by your own name; but then, the concern that I would see is that it wouldn't be the same name for the Spanish translation in H-uru and in MOUL(a)...

For the images, you can keep the ones that you need, and delete the other.

**Comment by Deledrius** [comment]
> I don't find how to "Amend" the commit in GitHub nor in Fork... If you could do it for me, please, it would be nice. 

In Fork, when you are staging a commit, there is a small checkbox just under the commit description area labeled "Amend".  

If checked, it will show the currently-staged changes as if they were part of the previous commit, and when committed it will *replace* the previous commit with the combined changes (that is, the previous commit plus whatever you have staged with "Amend" as a single commit).  Clicking "Amend" will automatically populate the commit description with the previous commit as well, allowing you to edit the contents if needed.

**Comment by TheScarFr** [comment]
> > I don't find how to "Amend" the commit in GitHub nor in Fork... If you could do it for me, please, it would be nice.
> 
> In Fork, when you are staging a commit, there is a small checkbox just under the commit description area labeled "Amend".
> 
> If checked, it will show the currently-staged changes as if they were part of the previous commit, and when committed it will _replace_ the previous commit with the combined changes (that is, the previous commit plus whatever you have staged with "Amend" as a single commit). Clicking "Amend" will automatically populate the commit description with the previous commit as well, allowing you to edit the contents if needed.
> 
> Note: Amending a commit this way only works on the most recent commit. If you need to edit other commits in a branch/PR it takes a bit more work, and can be scary if you're not experienced with it.

Thanks for the explanation. :) So yeah, I can't do it...

**Comment by Hoikas** [comment]
> Why are the email addresses needed?

They are needed to properly link the contributor's name with their GitHub account (if they have or will have one). Considering that none of us are being paid with real money for this work, the only form of payment anyone really gets for their contributions is the credit for their work. It's very important to us as an organization that we properly credit people who make contributions. I will make the change to the commit once we have either email addresses or word that they don't wish to provide one for Riccardo Salerano and Salvo Ludas.

> For the Spanish translation, If you want to replace "Brazalete" by "Pulsera", you will have to replace "Salvo Ludus" by your own name

I will get credit on the commit that changes the translation. Salvo Ludas will still need to be credited for the original (albeit incorrect) translation.

> the concern that I would see is that it wouldn't be the same contributor name for the Spanish translation in H-uru and in MOUL(a), and also not the same Spanish translation in the wardrobe.

Someone will have to mirror the correction to the MOULa repository. Does MOULa even support changing the language yet?

> It's just a little word without any real importance, that players generally don't pay attention to...

If that were the case, it would be ok for me to rename the "Ferry Terminal" to the "Fairy Terminal". Those are two different things, just like pulsera and brazalette are two different things. We need to use the correct words.

**Comment by TheScarFr** [comment]
@Hoikas :
Riccardo Salerano and Salvo Ludus aren't on GitHub.
I don't want to add my email address, "TheScar(.fr)" is enough for me.

**Comment by Deledrius** [comment]
> @Hoikas : Riccardo Salerano and Salvo Ludus aren't on GitHub. I don't want to add my email address, "TheScar(.fr)" (which is my OOC avatar name) is enough for me.

The commits you've made in this branch have already been assigned one by Github, which is used to identify you with your account.

**Comment by Hoikas** [comment]
We cannot proceed until we have a response from Riccardo Salerano and Salvo Ludus, I'm afraid.

**Comment by TheScarFr** [comment]
@Hoikas I sent them the request by private messages, I'm waiting for their answers.

**Comment by EhrenCG** [comment]
Despite lack of much use, I assume my account here was found?

I didn't make the initial bracelet, but for the MOUL converted version, I figured out how to unscramble the male avatar's hand and get the bracelet to actually stick to the wrist.

**Comment by TheScarFr** [comment]
```
- Development:		Sirius > His email address isn't written on his profile > https://github.com/Jrius
- Convertion/Edit:		TheScar(.fr) <betweenthecleftandthescar@gmail.com>
- Convertion fixes:		Ehren <EhrenCG@hotmail.com>
- Translation:
	- English/French:	TheScar(.fr)
	- German:		dgelessus <dgelessus@users.noreply.github.com>
	- Italian:			Riccardo Salerano <rickcourriel@protonmail.com>
	- Spanish:			Salvo Ludus <salvoludus@live.com>
```

**Review by Hoikas — APPROVED** [review]
I've straightened out all of the authorship information and removed the tray merge commits (and fixed the unattributed change included in a merge commit). Thanks everyone for the new clothing item, it looks nice in game!

**Comment by Deledrius** [comment]
FYI @TheScarFr, I am not @dgelessus.

**Comment by TheScarFr** [comment]
> FYI @TheScarFr, I am not @dgelessus.

Oops, I'm really sorry...

**Comment by Deledrius** [comment]
No worries, just making note for future reference in case you confused us rather than it being a typo.

**Comment by TheScarFr** [comment]
Yeah it was a typo...

**Pull request merged**